### PR TITLE
Testing counter comments

### PR DESCRIPTION
**Pull request by Luiscarlosvd**
*Features:*
- The function for the comments counter is covered with unit tests using Jest.
- There are tests for all edge cases.
- Fix linters.

*Author:*
@Luiscarlosvd

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
+    "jsdom": "^22.0.0",
```
Comment (mrcbq) [inline package.json:38]: This library is not helpful in our project, we are not using it. It may be better to remove it from this file.

**Review by mrcbq — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Luiscarlosvd ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:

## [Optional] suggestions:
Hello please take into account this change before merging your code.

Thank you for your contribution to the project, and I appreciate the effort you've put into this feature.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
